src/playlist-loader.js

```diff
@@ -537,6 +537,8 @@ export default class PlaylistLoader extends EventTarget {
         setupMediaPlaylists(this.master);
         resolveMediaGroupUris(this.master);
 
+        // This will trigger an existing child loader to be loaded
```

Comment:
when I said "loaded" here I meant `load()` is called, but I think your explanation is a bit clearer

Reply:
oh, also this is meant to be about the child playlist loader's `loadedplaylist` event that calls `load()`

Reply:
changing this to:

>>>If this is a child loader, this will trigger the child loader to make a request for the child manifest

Reply:
but this is within `if (parser.manifest.playlists)` which would only evaluate to true if it parsed a master playlist, which a child loader would never have, so it would never be a child loader at this point

Reply:
_facepalms_

maybe this was the wrong one...